requirements-override.txt

```diff
@@ -1,2 +1,3 @@
 bigeye-airflow==0.1.34
 bigeye-sdk==0.4.81
+apache-airflow-providers-cncf-kubernetes==8.3.4
```

Comment:
Tested this locally and it doesn't look like it will affect any other packages/dependencies. 👍

Reply:
Also, looking forward to when we upgrade Airflow itself, this override will still be necessary for Airflow 2.9.2 since it defaults to `apache-airflow-providers-cncf-kubernetes` version 8.3.1, but it can be removed once we upgrade to Airflow 2.10.2 since that defaults to `apache-airflow-providers-cncf-kubernetes` version 8.4.1.